### PR TITLE
fix: Session and Exhibitor Form Simplify Slide Form Element

### LINKED ISSUE #7000: Session and Exhibitor Form: Simplify Slide Form Element
In the past we had decided to offer an option to either upload slides or link them from somewhere else. It turns out that this makes things more complicated. We cannot validate slides and link locations of slides might change.

Therefore change the slide form areas in the session form and the exhibitor form to only provide the option to upload slides.
Also change the following:
* Do not show the API link of slides anymore, instead show a more standard display with the slide name similar to other services with this feature

For existing slides that are on external pages, please use a script to download these slides and import them into the system and show them as uploaded slides.

### PR DESCRIPTION
**Pull request by ranjitkshah**
Fixes #7000 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Same comment as in previous PR:

1. Do not show preview in the form itself. Do not show the slides itself on the form.
2. Where does the comment regarding the video in the slides section come from - "Slides uploaded..."? Please delete the text. It does not make sense.
3. Change form text "Slide" to "Slides"
4. Change "Slides URL" to "Slides"

Unfortunately, you did not pay attention to details. Would be great if you could help to make things easier for reviewers.

**Comment by mariobehling** [comment]
Even though not every single point was stated in the issue, developers should check the logic of the section they work on, e.g. if you take out the URL option then the heading should not be called "Slides URL" anymore. Lets keep this project fun and think things through when we collaborate here. Thank you.

**Comment by MeghalBisht** [comment]
> 1. Do not show preview in the form itself. Do not show the slides itself on the form. Only show it on the output page.

It was already done and reviewed be me and @maze-runnar in the previous PR.

> 2. Where does the comment regarding the video in the slides section come from - "Slides uploaded..."? Please delete the text. It does not make sense.

This was not mentioned by you in the previous PR, @ranjitkshah please follow up here!

> 3. Change form text "Slide" to "Slides"
> 4. Change "Slides URL" to "Slides"

Again, not mentioned in the previous PR. It does make sense somehow, because after the slides are uploaded we see the generated URL. Anyways, @ranjitkshah Please follow up here as well.

> Unfortunately, you did not pay attention to details. Would be great if you could help to make things easier for reviewers.

@mariobehling The previous PR was not randomly merged after 2 approved reviews. The only problem with the last merged PR was that it was directed into the `development` branch. Hence it was reverted, I asked @ranjitkshah to generate a new PR for the `testing` branch. I believe he forgot to edit the PR description and attached the previous images which is acceptable considering that the PR already passed the checks before. 
Fortunately, we are attending the PRs with care and two of us are testing even the smallest opened PR just to avoid any wrong code being pushed, which takes a significant amount of time and work. Jumping onto conclusions by just seeing the wrong image attached is not a correct way to judge the work we all are doing. Infact, we would be the last people to take this project as fun, we are now accountable for the work getting merged here. Any wrong code being pushed would be & should be blamed on our heads. Thanks!

File: app/templates/components/widgets/forms/file-upload.hbs [hunk 1]
```diff
@@ -10,13 +10,9 @@
   {{else}}
     <div class="ui grid">
       <div class="column sixteen wide">
-        <Input @type="url" @change={{action "fileSelected" "url"}} />
         <span class="text muted">
           ({{t 'Slides uploaded here are not integrated in the video system. For video conferences please upload your slides into the video session directly as well.'}})
         </span>
```
Comment (maze-runnar) [inline app/templates/components/widgets/forms/file-upload.hbs:15]: remove this text, as mario said.

**Comment by ranjitkshah** [comment]
Screenshot from 2021-06-09 00-45-40
Screenshot from 2021-06-09 00-49-31

**Comment by ranjitkshah** [comment]
> 3. Change form text "Slide" to "Slides"

do i need to run migrations for this @MeghalBisht @maze-runnar

**Comment by MeghalBisht** [comment]
> do i need to run migrations for this @MeghalBisht @maze-runnar

Ah, yes. Good point

**Review by MeghalBisht — APPROVED** [review]

**Review by maze-runnar — DISMISSED** [review]

File: app/templates/components/forms/session-section.hbs [hunk 2]
```diff
@@ -2,7 +2,11 @@
   {{#if field.isIncluded}}
     <div class="field">
       <div class="inline">
-        <label class="{{if field.isRequired 'required'}}" for="name">{{field.name}}</label>
+        {{#if (eq field.name 'Slide')}}
+          <label class="{{if field.isRequired 'required'}}" for="name">Slides</label>
+        {{else}}
+          <label class="{{if field.isRequired 'required'}}" for="name">{{field.name}}</label>
+        {{/if}}
```
Comment (sachinchauhan2889) [inline app/templates/components/forms/session-section.hbs:9]: Now no need of migrations.

**Review by sachinchauhan2889 — APPROVED** [review]

File: app/templates/components/forms/session-section.hbs (hunk 2 above)
Comment (maze-runnar) [reply app/templates/components/forms/session-section.hbs:9]: 🔥

**Review by maze-runnar — APPROVED** [review]

**Review by MeghalBisht — DISMISSED** [review]

**Review by sachinchauhan2889 — APPROVED** [review]

**Pull request merged**